src/MMKV.ts

```diff
@@ -1,4 +1,6 @@
 import { createMMKV } from './createMMKV';
+import { createMockMMKV } from './createMMKV.mock';
+import { isJest } from './PlatformChecker';
```

Comment:
having an export for 1 LOC seems unnecessary. also, file is capitalized despite not being a react component or class.

Reply:
Understandable, but I feel this is cleaner than having `process.env.JEST_WORKER_ID != null` inside the `MMKV` constructor.

Re filename - I viewed `PlatformChecker` as more of a "sub-domain" and not a generic file like `helpers`/`utils` hence the capitalisation; not a strong preference though, happy to change it to something else if that's the concensus.

Reply:
I like it the way it is, let's leave it.